mods/ra/maps/allies-01/allies01.lua

```diff
@@ -19,6 +19,16 @@ end
 SendJeeps = function()
 	Reinforcements.Reinforce(player, JeepReinforcements, InsertionPath, DateTime.Seconds(2))
 	Media.PlaySpeechNotification(player, "ReinforcementsArrived")
+
+	-- TESTCASE
```

Comment:
Ah, nevermind.  I didn't realize that you got jeep reinforcements later in the mission.